kustomize/kustomization.yaml

```diff
@@ -23,3 +23,7 @@ components:
 # - components/network-policies
 # - components/service-accounts
 # - components/spanner
+# - components/container-images-tag
+# - components/container-images-tag-suffix
+# - components/container-images-registry
```

Comment:
**Praise:** 
Thanks for putting these in the working right order (instead of in alphabetical order)!
```
- components/container-images-tag
- components/container-images-tag-suffix
- components/container-images-registry
```